aspnetcore/blazor/security/webassembly/additional-scenarios.md

```diff
@@ -1061,6 +1061,17 @@ The placeholder `{APP ASSEMBLY}` is the app's assembly name (for example, `Blazo
 
 For more information, see <xref:grpc/browser>.
 
+## Build a custom version of the Authentication.MSAL JavaScript library
+
+If an app requires a custom version of the [Microsoft Authentication Library for JavaScript (MSAL.js)](https://www.npmjs.com/package/@azure/msal-browser), perform the following steps:
+
+1. Confirm the the system has the latest developer .NET SDK or obtain and install the latest developer SDK from [.NET Core SDK: Installers and Binaries](https://github.com/dotnet/installer#installers-and-binaries). Configuration of internal NuGet feeds isn't required for this scenario.
+1. Fork and clone or download a ZIP archive of the [dotnet/aspnetcore GitHub repository](https://github.com/dotnet/aspnetcore).
```

Comment:
You can link to the [setup docs](https://github.com/dotnet/aspnetcore/blob/master/docs/BuildFromSource.md) for the repo. Something like "Setup the repo for development per the docs at"